format-blob/src/test/java/io/cdap/plugin/format/blob/input/BlobInputFormatProviderTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2018-2019 Cask Data, Inc.
```

Comment:
fix copyright year. Any new file should have just the current year.

Reply:
Changed to 2022